src/backend/storeManagers/gog/games.ts

```diff
@@ -558,6 +559,13 @@ export async function launch(
 
     const { bin: wineExec, type: wineType } = gameSettings.wineVersion
 
+    if (wineType === 'proton') {
```

Comment:
Should this be?

```suggestion
    if (isUmuSupported(wineType)) {
```

to avoid the extra API request if umu is not enabled

maybe this if can also be moved somewhere else? I see it's the same code in the 3 stores